ansible/roles/iroha-docker/defaults/main.yml

```diff
@@ -48,12 +48,12 @@ iroha_volume_dir: ""
 
 iroha_docker_image_pull: true
 iroha_docker_image: 'hyperledger/iroha'
-iroha_docker_tag: '1.1.0'
+iroha_docker_tag: 'develop'
 
 # Use this parameter to start Iroha with additional command line option.
 # Example, enabling WSV Reuse:
-# iroha_docker_commands:
-#  - '--reuse_state'
+iroha_docker_commands:
+  - '--drop-state'
```

Comment:
it is a bad idea to update `default` value. Imagen the situation you have production, you pulled new ansible role (e.g updated submodule), and in your next update you droped statet